app/scripts/lib/metrics.js

```diff
@@ -64,40 +69,42 @@ define([
   function Metrics (options) {
     /*eslint complexity: [2, 18] */
```

Comment:
According to ESLint, our complexity jumped from `18` to `23` here.

**Workaround:**
```js
/*eslint complexity: [2, 23] */
```


```
app/scripts/lib/metrics.js
   69:2  error  Function 'Metrics' has a complexity of 23       complexity
```